packages/types/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@comunica/types",
+  "version": "1.19.2",
+  "description": "Typings module for Comunica",
+  "lsd:module": false,
```

Comment:
Or just removing this line would also be ok :-)

Reply:
Will do!

Reply:
Done!